apps/docs-snippets/src/guide/contracts/logs.test.ts

```diff
@@ -29,7 +29,7 @@ describe(__filename, () => {
     expect(new BN(logs[0]).toNumber()).toBe(value1);
     expect(logs[1]).toBe(value2);
     expect(logs[2]).toBe(value3);
-    expect([logs[3], logs[4], logs[5]]).toEqual(value4);
+    expect(logs[3]).toEqual(value4);
```

Comment:
This is a fix and is how it's supposed to work. If you look at the corresponding sway file, you'll see that the fourth log entry is an array, which is what it is here now with `logs[3]`.